package.json

```diff
@@ -11,9 +11,10 @@
     "lint:eslint": "eslint packages",
     "lint:mrl": "mrl check",
     "lint:prettier": "prettier --check .",
+    "postinstall": "husky install",
     "preinstall": "npx only-allow pnpm",
-    "prepare": "lerna run build && husky install",
-    "test": "pnpm run lint && lerna run test && lerna run --scope @turf/turf last-checks"
+    "prepublish": "lerna run build && lerna run --scope @turf/turf last-checks",
+    "test": "lerna run test"
```

Comment:
Just a couple thoughts here:

1) it dropped `pnpm run lint`
2) I don't remember exactly why but I think last-checks depends on `lerna run test`?

What do you think about making a new script entry:
`"ci": "lerna run build && lerna run lint && lerna run test && lerna run --scope @turf/turf last-checks"

And then just calling this new script explicitly in the .github/*.yml files?
We could also break build / lint / (test & last checks) into 3 different scripts for more granular timings on CI. In general I think we can get away from some of the magic script names that make pnpm do things at certain lifecycle moments.

Reply:
I looked into this some more and `last-checks` only depends on `last-checks:testjs` to run before `last-checks:example` and it doesn't actually depend on the other packages, so that seems safe to move. I'm not actually sure why it lives in its own step, I'll make a PR cleaning that up.

Reply:
https://github.com/Turfjs/turf/pull/2690 cleans up some last-checks

Reply:
I think I'm leaning towards the following:

package.json scripts:
```
"build": "lerna run build",
"lint": "lerna run lint",
"test": "lerna run test"
```

github workflows:
`run: pnpm build` added after `run: pnpm install --frozen-lockfile` everywhere
`run: pnpm lint` added before `run: pnpm test` everywhere

This also will cause us to need to make changes in the CONTRIBUTING.md file
- `Local Setup` no longer requires `pnpm test`
- `Development tips` and `Pull Request Checklist` can probably also recommend running `pnpm lint`